RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -366,6 +366,16 @@ class PumpOpsSynchronous {
                             events.insert(TimestampedHistoryEvent(pumpEvent: event, date: date), atIndex: 0)
                         }
                     }
+                    
+                    if let alarm = event as? PumpAlarmPumpEvent {
+                        switch alarm.alarmType {
+                        case .BatteryDepleted, .BatteryOutLimitExceeded, .DeviceReset:
+                            print("Found clock loss in pump history.  Ending history fetch.")
```

Comment:
Yeah, I was wondering if that's what happened with @loudestnoise's pump. 

The reset to zero is an interesting idea, but wouldn't it cause bad interpretation of older events if there was also a valid time change interval?